idea/common/src/enums/code-status.enum.ts

```diff
@@ -1,4 +1,5 @@
-export enum CodeStatus  {
+export enum CodeStatus {
   ACTIVE = 'Active',
   INACTIVE = 'Inactive',
+  UNKNOWN = 'Unknown',
```

Comment:
Please do not forget to check is need it to create a database migration `file` by `code entity`

Reply:
btw, thanks